### PR TITLE
new and added

### PR DESCRIPTION
**Pull request by davesaddle**
Added user, templates, Font Awesome  
commented on Mobile detecting..

### CODE REVIEW

**Comment by scottcwilson** [comment]
Thank you for your contribution!

File: content/user/template/font_awesome.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+title: Font Awesome 
+description: Using Font Awesome from local and remotely.
+category: template
+weight: 10
+---
+
+Font Awesome 5 was a full rewrite and the Unicode charter set no longer work as they did with version 4. With version 4 using the Unicode like this `&#xf217; Add to Cart ` worked when creating input buttons. With version 5 you now have to do it like this `<i class=”fa”>&#xf217;</i> Add to Cart` or use it within a class as
```
Comment (drbyte) [inline content/user/template/font_awesome.md:8]: Thanks for the contribution!

I'd like to see this updated to not have the tone of "Welcome to this new topic ... the first thing we're gonna talk about is how we don't do it the old way anymore, and first let us explain the old way"

Preferably the tone should be somewhat imperative: "this is how to do it". Perhaps further down leave a section for how to convert the old, or if you still want to explain how to do the old way, do it at the bottom.

Comment (davesaddle) [reply content/user/template/font_awesome.md:8]: OK.. based on how I've always used it. I can change the lead in to the old way I used and now using... add a fixme to the bottom for others to add how they use it.. new at this, but I know there are more then one way to code FA.

Comment (scottcwilson) [reply content/user/template/font_awesome.md:8]: One small suggested tweak: at the top in the front matter, add 

type: codepage

below the weight.  This lets you go really wide, which is nice for these code samples.

**Pull request merged**